### PR TITLE
Update Docs/Storybook/VRT for new Button API

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
- Action Button
  - add `variant="black | white"` examples
- Button
  - normalize story listings
  - add `variant="accent | black | white"`
  - add `treatment="fill | outline"`
- Split Button
  - normalize story listing
  - deduplicate story listing
  - add `variant="accent"` 

## Motivation and context
Should be easy to understand the available capabilities of Action Button, Button, and Split Button elements.

## How has this been tested?

-   [ ] _Test case 1_
    1. Go [here](https://d3ea2b33f0029f78f894b11dde61060a--spectrum-web-components.netlify.app/review/#ActionButtonStories/toggles.png)
    2. Review the added and removed tests
    3. Updated tests _should_ wash out when we update the Golden Image Cache in #2004 

## Types of changes
-   [x] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist
-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.

### CODE REVIEW

**Comment by castastrophe** [comment]
Summary from our call 12/17 - these design updates look solid; improved accessibility but low risk of a breaking design change.

**Review by castastrophe — APPROVED** [review]
This is looking really solid, I like the way you approached this update a lot.

File: packages/button/stories/index.ts [hunk 1]
```diff
@@ -15,9 +15,67 @@ import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
...
+                'secondary',
+                'negative',
+                'overBackground',
+                'black',
+                'white',
+            ],
+        },
+    },
+    treatment: {
+        name: 'tratment',
```
Comment (hunterloftis) [inline packages/button/stories/index.ts:62]: Is this what's causing all the Button/*/Outline/Icon-Size-Overridden stories to use treatment='fill'?

```suggestion
        name: 'treatment',
```

**Review by hunterloftis — APPROVED** [review]
👍  Looks good, especially for such a big diff. Noticed only the one issue.

File: packages/button/stories/index.ts (hunk 1 above)
Comment (Westbrook) [reply packages/button/stories/index.ts]: 😱 so many time little things to catch! Added that arg to the icon size stories, thanks.

**Pull request merged**